sklearn/multioutput.py

```diff
@@ -538,7 +538,7 @@ class labels for each estimator in the chain.
 
     See also
     --------
-    RegressorChain
+    RegressorChain, MultioutputClassifier, MultioutputRegressor
```

Comment:
MultioutputRegressor is not relevant here.

I also think you should describe how it differs from this in very few words. Take a look at other see alsos.